MapboxVisionTests/DeviceCheckerTests.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Created by Dersim Davaod on 5/8/19.
+//  Copyright © 2019 Mapbox. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import MapboxVision
+
+class DeviceCheckerTests: XCTestCase {
+    func testIsHighPerformanceDeviceReturnsFalseOnIphone7PlusOrLower() {
+        // Given iPhone 5s (GSM)
+        // When // Then
```

Comment:
@dersim-davaod what does `// When // Then` mean?

Reply:
We agreed to use [`Given-when-then model`](https://martinfowler.com/bliki/GivenWhenThen.html) in our unit tests. Corresponding section should be marked with comments. In that case appropriate comments were left for consistency.